examples/js/archive.js

```diff
@@ -57,8 +65,108 @@ function extractKey() {
   }
 }
 
-let imageCount = 0;
-let fetchedFrom;
+// <---------- pagination start
```

Comment:
This looks good, but it is a lot of code! What do you think about how this code might be compartmentalized -- could this be moved into a separate module, perhaps included with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules ? 

How might that also affect variable naming and scope - could we reduce the number of variables we track in the main scope, by moving them into their own file and module? It's just a lot of code here, and we aren't doing JS tests to protect it, so it seems like it could be tough to maintain longer term!

Reply:
Alright @jywarren I would look into how feasible using modules
Thank you for your review!